src/ontology/mondo-edit.obo

```diff
@@ -93339,12 +93344,16 @@ xref: SCTID:200779005 {source="DOID:3310"}
 xref: SCTID:24079001 {source="DOID:3310", source="EFO:0000274", source="MONDO:equivalentTo", source="https://orcid.org/0000-0002-9584-9618"}
 xref: SCTID:90823000 {source="EFO:0000274"}
 xref: UMLS:C0011615 {source="MEDGEN:41502", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
+xref: UMLS:C1864155 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:350353"}
 is_a: MONDO:0002406 {source="DOID:3310", source="EFO:0000274", source="MONDO:Redundant", source="NCIT:C3001"} ! dermatitis
+is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditary disease
```

Comment:
remove this

this will be an unusual case where we have a Mondo term that is equiv to OMIM but it is not hereditary

Reply:
fixed